poker.js

```diff
@@ -87,49 +87,76 @@ for (let i = 0; i < sorted.length - 1; i++) {
 const highCard = (rank) => {
   let highestCard = 0;
   for (let elem of rank) {
-    highestCard < elem ? highestCard = elem : highestCard;
+    highestCard < elem ? highestCard = Number(elem) : highestCard;
   }
   return highestCard;
 };
 
+const PlayerHand = function (handType, strength) {
+  this.hand = handType;
+  this.highCard = strength;
+}
+
```

Comment:
Lets go with a more functional approach I'd say..
`new` keyword isn't needed anymore
```suggestion
const PlayerHand = (hand, highCard) => ({
  hand,
  highCard,
});

```

Reply:
Done!